google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/StreamWriter.java

```diff
@@ -173,6 +174,11 @@ public class StreamWriter implements AutoCloseable {
    */
   private final AtomicLong inflightWaitSec = new AtomicLong(0);
 
+  /*
+   * A String that uniquely identifies this writer.
+   */
+  private final String writerId = UUID.randomUUID().toString();
```

Comment:
Multiplexing will work underneath. This is purely a writer ID. Since I also return the StreamName, so maybe it doesn't need to include the stream identity. Multiple writers can work on same stream for all types of streams.